bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/Client.scala

```diff
@@ -48,6 +48,9 @@ trait Client extends BitcoinSLogger with StartStopAsync[BitcoindRpcClient] {
   def version: BitcoindVersion
   protected val instance: BitcoindInstance
 
+  protected def walletExtension(walletName: String): String =
+    s"/wallet/$walletName"
```

Comment:
Should this pattern match on the version? I think it's probably a good idea in case extensions change in the future, OR we can throw an exception here if it's an older version of bitcoind that doesn't support multi wallet?

Reply:
Actually thinking about this more, maybe we want to just incorporate the wallet  name inside of `BitcoindRpcAppConfig`?

Reply:
I thought about doing it this way but the problem is that we then need a `BitcoindInstance` and thus client for every wallet we want to use which could get really annoying.